spec/events/inventory_aggregate_spec.rb

```diff
@@ -381,11 +381,14 @@
     end
 
     it "should process a kit allocation event" do
-      kit = FactoryBot.create(:kit, :with_item, organization: organization)
+      params = FactoryBot.attributes_for(:kit)
+      params[:line_items_attributes] = [
+        {item_id: item1.id, quantity: 10},
+        {item_id: item2.id, quantity: 3}
+      ]
+
+      kit = KitCreateService.new(organization_id: organization.id, kit_params: params).call.kit
```

Comment:
Is there a way for us to just update the factory itself to call `KitCreateService`? E.g. by overriding `to_create`? https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#custom-methods-to-persist-objects

If not, can we create a reusable method to do this work, allowing you just to pass in the LineItems you want to create, and provide a sane default if we don't care what they are?

Reply:
I considered overriding `to_create` but I don't think it'll work because the way it works right now KitCreateService creates its own instance of kit and I don't think it's worth the time to rejig it so that it can accept FactoryBot's instance, since it will be changed in the kit refactoring.

I created a helper method under `spec/support/kit_helper.rb` in [730371a](https://github.com/rubyforgood/human-essentials/pull/4585/commits/730371a14689e5a8a7c1bab825f4df08fc3e279e)